ducktape/tests/loader.py

```diff
@@ -154,21 +155,23 @@ def import_modules(self, file_list):
 
             # Try all possible module imports for given file
             path_pieces = f[:-3].split("/")  # Strip off '.py' before splitting
+            success = False
             while len(path_pieces) > 0:
                 module_name = '.'.join(path_pieces)
                 # Try to import the current file as a module
                 try:
                     module_list.append(importlib.import_module(module_name))
                     self.logger.debug("Successfully imported " + module_name)
+                    success = True
                     break  # no need to keep trying
                 except Exception as e:
-                    self.logger.debug("Could not import " + module_name + ": " + e.message)
-                    self.logger.debug(
-                        "   But it's ok: at most one import should work per module, so import failures are expected.")
                     continue
                 finally:
                     path_pieces = path_pieces[1:]
 
+            if not success:
+                self.logger.debug("Unable to import %s" % f)
```

Comment:
Hm, actually this change was almost entirely the reason for this tiny PR.

For any given file, there are a bunch of import attempts until one works, or until all attempts fail.
We really just want to know whether an import succeeded overall (at least on import path worked), or if it failed overall (all import paths failed for that file).

Previously the logging for this was quite noisy, since each file would have many failed import attempts, even though the overall import of the given file was successful, and I found it hard to see when a file truly failed to import.